layouts/list.algolia.json

```diff
@@ -1,9 +1,9 @@
 {{- $.Scratch.Add "index" slice -}}
 {{- $section := $.Site.GetPage "section" .Section }}
 {{- range .Site.AllPages -}}
-  {{- if or (and (.IsDescendant $section) (and (not .Draft) (not .Params.private))) $section.IsHome -}}
+  {{- if or (and (.IsDescendant $section) (and not .Params.private)) $section.IsHome -}}
     {{ if ( .Params.company ) and ( .Params.company "!=" "") and ( .Params.company "!=" null) }}
```

Comment:
Place the `null` in quotes and it should work.

Reply:
The `null` was throwing it off. The purpose of this line was to make sure we weren't generating any blank cards (previous issue). Seems leaving `null` as is was giving errors. We could probably remove the check for `null` since it probably defeats the purpose of putting it in quotes.

Sorry. Seems to be a bug that I added earlier in my last PR.

Reply:
oh gotcha - no worries!